### PR TITLE
Fix issues, grammar, and typos

### PR DESCRIPTION
**Pull request by cadamini**
Replaces https://github.com/jobream/List-of-Learning-Resources/pull/132 (I had accidentally deleted my fork)

I spotted some issues:
- an extra image was breaking a heading position
- typos 
- minor grammar issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -163,6 +164,10 @@ Pull requests are welcome 👍. For major changes, please open an issue first to
 ------------------------------------
 ## Contributors ✨
 
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification for recognizing project contributors. 
+
+Contributions of any kind welcome!
```
Comment (cadamini) [inline README.md:169]: reviewer comment: I think this is better located on top of the section. Let me know if I need to revert this.

File: README.md [hunk 2]
```diff
@@ -115,7 +116,7 @@ complete beginners.
 
 ## Extras : Books and Video Courses! :
 
-<img src="https://cdn-icons-png.flaticon.com/128/3730/3730087.png" height="60px"> <img src="https://image.flaticon.com/icons/png/128/2096/2096018.png" height="60px">
```
Comment (cadamini) [inline README.md:118]: reviewer comment. Only one image exists. Removing the 2nd one fixes the heading position.

File: README.md [hunk 3]
```diff
@@ -92,8 +92,9 @@ complete beginners.
 - [ziishaned-repository](https://github.com/ziishaned/learn-regex) - This is a resource for learning Regular Expressions in many languages.
 - [Regex tutorials](http://regextutorials.com/) - Provides both theory and exercises to learn Regular Expressions.
 
-### 4. Learn Solidity Samrt Contract with hardaht - Step by Step code and Video Links : 
+### 4. Learn Solidity Smart Contract with hardhat - Step by Step code and Video Links :
```
Comment (cadamini) [inline README.md:95]: reviewer comment: typos

File: README.md [hunk 4]
```diff
@@ -102,11 +103,11 @@ complete beginners.
...
-- [Hyderabad Developer Community](https://www.facebook.com/groups/HyderabadDeveloperCommunity) - This is community of Hyderabad, Pakistan for developers.           
-- [Code With Concepts](https://www.facebook.com/groups/codewithconcepts) - This is community of teaching programming to newbie.
-- [Mentors Without Borders](https://www.facebook.com/MentorsWB) - This is community of giving mentorship for free to students.
+- [Freecodecamp Karachi](https://www.facebook.com/groups/free.code.camp.karachi) - Freecodecamp community from Karachi, Pakistan.
+- [Golang Pakistan](https://www.facebook.com/groups/2547735885480597) - Golang community from Pakistan.                         
+- [Hyderabad Developer Community](https://www.facebook.com/groups/HyderabadDeveloperCommunity) - Hyderabad community from Pakistan for developers.           
+- [Code With Concepts](https://www.facebook.com/groups/codewithconcepts) - Community of teaching programming to newbie.
+- [Mentors Without Borders](https://www.facebook.com/MentorsWB) - Community of giving mentorship to students for free.
```
Comment (cadamini) [inline README.md:110]: reviewer comment: grammar

File: README.md (hunk 2 above)
Comment (CodingSpecies) [reply README.md:118]: Yes, that would be good @cadamini . 
Thanks.

**Review by CodingSpecies — APPROVED** [review]
Thank you ever so much @cadamini for your useful contributions!

**Pull request merged**